test-py/test_backends_fontra.py

```diff
@@ -136,3 +136,23 @@ async def test_statusFieldDefinitions(writableFontraFont):
     await writableFontraFont.putCustomData(statusTestData)
 
     assert statusTestData == await writableFontraFont.getCustomData()
+
+
+async def test_getGlyphSourceStatusCode(testFontraFont):
```

Comment:
These tests are redundant, as there is no special support needed for customData or status codes. Please remove these tests.

Reply:
done.